src/pxl_scripts/px/mongodb_data/mongodb_data.pxl

```diff
@@ -0,0 +1,116 @@
+# Copyright 2018- The Pixie Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+''' MongoDB Data Tracer
+
+Shows the most recent MongoDB messages in the cluster.
+'''
+import px
+
+
+def mongodb_data(start_time: str, source_filter: str, destination_filter: str, num_head: int):
+
+    df = px.DataFrame(table='mongodb_events', start_time=start_time)
+    df = add_source_dest_columns(df)
+
+    # Filter out entities as specified by the user.
+    df = df[px.contains(df.source, source_filter)]
+    df = df[px.contains(df.destination, destination_filter)]
```

Comment:
This could be my knowledge gap, but what happens when the filter string is empty? Or is the default something like `*`?

Reply:
I believe the default source filter is already to set to empty so it would display all origins of mongo requests

Reply:
That's correct. Another thing I wanted to mention was that was something we didn't have a compiler optimization for until recently to prune "empty contains" calls ([pixie#1679](https://github.com/pixie-io/pixie/pull/1679))